eng/generators.targets

```diff
@@ -94,5 +114,15 @@
     </PropertyGroup>
   </Target>
 
+  <!-- System.Text.RegularExpressions -->
+  <Target Name="ConfigureRegexGenerator"
```

Comment:
As mentioned in a comment above, please do remove this target as we should use conditional compilation instead in case we needed this.